jQuery/index.html

```diff
@@ -1,10 +1,25 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+
     <title>Honey Headquarters</title>
   </head>
   <body>
-    Hello wolr
+    <div id="hh-menu">
+      <h1>Honey Headquarters</h1>
+
```

Comment:
an interesting way to structure labels with form elements can actually be implicit rather than explicit. You're explicitly declaring that a `label` is `for` another element. (which is valid) but you can also do this:

```
<label class="menu-label">
  <select id="character-select"></select>
</label>
```

The class on this label isn't necessary, just an idea. but semantically this is still valid, as the form element is __implicitly__ targeted by the label by virtue of being contained inside of it.